test/e2e/framework/framework.go

```diff
@@ -60,7 +60,7 @@ type Framework struct {
 	BaseName string
 
 	Client        *client.Client
-	Clientset_1_2 *release_1_2.Clientset
+	Clientset_1_5 *release_1_5.Clientset
 	Clientset_1_3 *release_1_3.Clientset
```

Comment:
Also delete Clientset_1_3 and StagingClient and replace their usages by 1.5 clientset?